docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.3'
 services:
   mysql:
-    image: mysql
+    image: mysql:5.7.21
```

Comment:
Are we ready for 5.7? I'm not up to date on our plans on this.  I just know that Accelerate production is still at 5.5 and that [AC-4618](https://masschallenge.atlassian.net/browse/AC-4618) is still TODO.

Reply:
@pbx we are already on 5.7, run `make db-shell` and you'll see.
I believe that without this lock the mysql installed version is 8, and we're not ready for _that_ one.

Reply:
@pbx we are not using mysql container on prod, this is only a development environment configuration at the moment. This PR doesn't change that (and again, we were already on 5.7 locally before this PR).